WordPress/Tests/Theme/DeregisterScriptUnitTest.inc

```diff
@@ -0,0 +1,9 @@
+<?php
+
+wp_enqueue_style( 'style', get_stylesheet_uri() ); // this is ok
+
+wp_enqueue_style( 'slider', get_template_directory_uri() . '/css/slider.css', false, '1.1', 'all' ); // this is ok
+
+wp_enqueue_script( 'script', get_template_directory_uri() . '/js/script.js', array( 'jquery' ), 1.1, true ); // ok
+
+wp_deregister_script( 'jquery' ); // this is bad, themes must not deregister scripts
```

Comment:
Maybe change some or all examples to use `wp_register_script/style()` ?
The current examples now made me question straight away whether the rule should cover just `wp_deregister_script()` or also `wp_dequeue_script()` (and no, of course not, parent theme vs child theme, but you can see the confusion).

For that matter - what about deregistering styles ? Should that be covered ?